packages/shared/src/modules3rdParty/cross-crypto/index.js

```
@@ -16,16 +16,17 @@ if (process.env.NODE_ENV !== 'production') {
+    globalThis.crypto.randomBytes || crypto.randomBytes;
   crypto.getRandomValues =
-    crypto.getRandomValues || global.crypto.getRandomValues;
+    crypto.getRandomValues || globalThis.crypto.getRandomValues;
 }
 crypto.$$isOneKeyShim = true;
-global.crypto.$$isOneKeyShim = true;
+globalThis.crypto.$$isOneKeyShim = true;
 
 if (process.env.NODE_ENV !== 'production') {
-  console.log('crypto-browserify polyfilled', crypto, global.crypto);
+  console.log('crypto-browserify polyfilled', crypto, globalThis.crypto);
```

ESLint

[error] 'globalThis' is not defined.

(no-undef)